ci/playbooks/multinode-customizations.yml

```diff
@@ -173,7 +173,7 @@
     - name: Ensure we know compute host keys
       ansible.builtin.shell:
         cmd: |
-          ssh-keyscan {{ hostvars[item].ansible_host }} >> ~/.ssh/known_hosts
+          ssh -o StrictHostKeyChecking=no {{ hostvars[item].ansible_user }}@{{ hostvars[item].ansible_host }} uptime
```

Comment:
I'd add a comment here saying why we run such of a dumb command here using SSH, clearly explaining that is to replace `ssh-keyscan` because of blabla.